plugins/DuskVerb/tools/tuner/vintage_minisweep.py

```
@@ -0,0 +1,121 @@
+        r = subprocess.run(
+            [sys.executable, str(REPO / "plugins/DuskVerb/tools/tuner/full_check.py"),
+             str(out_dir), str(ANCHOR), "--name", "BH-mini", "--category", "Halls"],
+            capture_output=True, text=True, timeout=60)
+    except subprocess.TimeoutExpired:
+        print(f"full_check timeout (>60s) for {out_dir}", file=sys.stderr)
+        return 999
+    for line in r.stdout.splitlines():
+        if "GATE(S) FAILED" in line:
+            try:
+                return int(line.split("✗")[1].strip().split()[0])
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)